### PR TITLE
3.2.1

### PR DESCRIPTION
**Pull request by superboyiii**

### CODE REVIEW

**Review by shargon — APPROVED** [review]

**Review by Liaojinghui — APPROVED** [review]

File: src/OracleService/OracleService.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Neo.ConsoleService" Version="1.2.0" />
-    <PackageReference Include="NeoFS.API" Version="3.1.0" />
+    <PackageReference Include="NeoFS.API" Version="3.1.0-CI01329" />
```
Comment (erikzhang) [inline src/OracleService/OracleService.csproj:9]: NeoFS.API should be updated too.

**Review by erikzhang — CHANGES_REQUESTED** [review]

**Review by Liaojinghui — DISMISSED** [review]

File: src/OracleService/OracleService.csproj (hunk 1 above)
Comment (superboyiii) [reply src/OracleService/OracleService.csproj]: > NeoFS.API should be updated too.

Done.

**Review by erikzhang — DISMISSED** [review]

**Comment by erikzhang** [comment]
Wait for #698

**Comment by superboyiii** [comment]
@erikzhang merge?

**Review by erikzhang — APPROVED** [review]

**Pull request merged**